internal/protoplugin/registry.go

```diff
@@ -253,6 +256,32 @@ func (r *registry) newMethod(svc *Service, md *descriptor.MethodDescriptorProto)
 	}, nil
 }
 
+// loadTransitiveFileDependencies registers services and their methods from "targetFile" to "r".
+// It must be called after loadFile is called for all files so that loadTransitiveFileDependencies
+// can resolve filedecscriptors op depdendencies
+func (r *registry) loadTransitiveFileDependencies(file *File) error {
```

Comment:
does this function need to return an `error` at all?

Reply:
My bad, forgot to pass a transitive error during a refactor. Good catch!